Sources/App/Controllers/BaseController.swift

```diff
@@ -0,0 +1,77 @@
+import Vapor
+import FluentProvider
+
+protocol BaseController {
+  associatedtype Resource: Model, JSONConvertible, ResponseRepresentable
+}
+
+enum Method {
+  case index
+  case show
+  case create
+  case destroy
+}
+
+extension BaseController {
+  
+  func addRoutes(to droplet: Droplet, namespace: String? = nil, only methods: [Method] = [.index, .show, .create, .destroy]) {
```

Comment:
Let's do the default methods only index and show.

Reply:
Fixed